src/main/scala/webservice/CromIamApiService.scala

```diff
@@ -14,132 +19,179 @@ trait SwaggerService extends SwaggerUiResourceHttpService {
 
 trait CromIamApiService extends Directives with SprayJsonSupport with DefaultJsonProtocol with RouteConcatenation {
 
-  def returnInternalServerError(msg: String)  = HttpResponse(InternalServerError, entity = msg)
+  implicit val system: ActorSystem
+  implicit def executor: ExecutionContextExecutor
+  implicit val materializer: ActorMaterializer
 
-  val workflowRoutes = queryRoute ~ queryPostRoute ~ workflowOutputsRoute ~ submitRoute ~ submitBatchRoute ~
+  def cromwellInterface: String
+  def cromwellPort: Int
+
+  // Header that Akka HTTP adds to every request on receive.
+  // We get an warning in logs if we don't strip it out before sending the request to cromwell
+  // HTTP header ‘Timeout-Access: <function1>’ is not allowed in requests
```

Comment:
Maybe add this link as further justification: ["https://github.com/akka/akka-http/issues/64"](https://github.com/akka/akka-http/issues/64)